code/src/components/Thoughts.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect, useCallback } from 'react'
+
+import AddThought from './AddThought'
+import ThoughtsList from './ThoughtsList'
+
+const thoughtsAPI = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'
+const heartIcon = <img alt='heart-icon' src={'assets/heart.png'} className='heart-icon'/>
+const emotions = [
+    'happy',
+    'sad',
+    'frustrated',
+    'angry',
+    'inspired',
+    'amused',
+    'confident',
+    'grateful',
+    'irritated',
+    'stressed'
+]

```

Comment:
I really like your 'all-the thoughts' idea.   I was wondering how you did that when I look att your Netlify.